tests/cbmc/proofs/s2n_stuffer_write_network_order/Makefile

```diff
@@ -0,0 +1,38 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+# this file except in compliance with the License. A copy of the License is
+# located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Expected runtime is 30 seconds.
+MAX_LENGTH = 3
```

Comment:
Try longer lengths, see if it gets too slow.

Reply:
Actually, in the new version, length > 8 exits immediatly. No need for a limit!

Reply:
We still need to specify the bound. CBMC errors out because it can statically determine the bound for the loop. But now we have the appropriate bound.